cypress/integration/seededFlows/adminFlows/users/manageRoles.spec.js

```diff
@@ -0,0 +1,132 @@
+// More on roles, https://admin.forem.com/docs/forem-basics/user-roles
+function openCreditsModal() {
+  cy.getModal().should('not.exist');
+  cy.findByRole('button', { name: 'Add role' }).click();
+
+  return cy.getModal();
+}
+
+function closeUserUpdatedMessage() {
+  cy.findByText('User has been updated').should('exist');
+  cy.findByRole('button', { name: 'Close' }).click();
+  cy.findByText('User has been updated').should('not.exist');
+}
+
+function checkUserStatus(status) {
+  cy.findByTitle('Current status').should((element) => {
+    expect(element.text().trim()).equal(status);
+  });
+}
+
+describe('Manage User Roles', () => {
+  describe('As an admin', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/adminUser.json').as('user');
+      cy.get('@user').then((user) => {
+        cy.loginUser(user);
+      });
+    });
+
+    describe('Changing Roles', () => {
+      beforeEach(() => {
+        cy.visit('/admin/users/2');
+      });
+
+      it('should change a role', () => {
+        checkUserStatus('Trusted');
+
+        cy.findByRole('button', { name: 'Trusted' }).should('exist');
+        openCreditsModal().within(() => {
+          cy.findByRole('combobox', { name: 'Role' }).select('Warn');
+          cy.findByRole('textbox', { name: 'Reason' }).type('some reason');
+          cy.findByRole('button', { name: 'Add' }).click();
+        });
+
+        cy.getModal().should('not.exist');
+        closeUserUpdatedMessage();
+
+        cy.findByRole('button', { name: 'Warned' }).should('exist');
+        checkUserStatus('Warned');
+        cy.findByRole('button', { name: 'Trusted' }).should('not.exist');
+      });
+
+      it('should remove a role', () => {
+        checkUserStatus('Trusted');
+        cy.findByRole('button', { name: 'Trusted' }).click();
+        cy.findByRole('button', { name: 'Trusted' }).should('not.exist');
+        checkUserStatus('Good standing');
+      });
+
+      it('should not remove the Super Admin role', () => {
```

Comment:
I believe we're looking to make it possible to remove the super admin role from a non-creator user in the future which is why I put this test in place. It should fail whenever that change is made.